examples/kfp-pipelines/superworkflows/ray/kfp_v2/superpipeline_noop_docId_v2_wf.py

```diff
@@ -62,6 +62,7 @@ def super_pipeline(
     p2_skip: bool = False,
     p2_noop_sleep_sec: int = 10,
     p2_ray_name: str = "noop-kfp-ray",
+    p2_ray_run_id_KFPv2: str = "",
```

Comment:
can it be p1_ parameter, so users don't have to add it for each step?

Reply:
Fixed. Thanks